bigquery_storage/tests/system/conftest.py

```diff
@@ -16,17 +16,196 @@
 """System tests for reading rows from tables."""
 
 import os
+import uuid
 
 import pytest
 
 from google.cloud import bigquery_storage_v1beta1
 
 
-@pytest.fixture()
+_ASSETS_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)), "assets")
+
+
+@pytest.fixture(scope="session")
 def project_id():
     return os.environ["PROJECT_ID"]
 
 
+@pytest.fixture(scope="session")
+def dataset(project_id):
+    from google.cloud import bigquery
+
+    bq_client = bigquery.Client()
```

Comment:
We should make a fixture for `bq_client` so that we don't have to auth multiple times. Likewise, we should probably make a fixture for `credentials` and use those for both the `bq_client` and `client` fixtures.

Reply:
I like the proposal, will refactor. :+1: